homeassistant/components/zwave_js/services.yaml

```
@@ -29,3 +29,37 @@ set_lock_usercode:
       example: 1234
       selector:
         text:
+
+set_config_parameter:
+  description: Set a configuration parameter to a Z-Wave device. You can provide either an entity_id or device_id.
```

If rebase onto the latest dev, it will provide a `name` as well.

Entity ID or Device ID isn't true I guess, as multiple and even an area can be targeted as well.

```suggestion
set_config_parameter:
  name: Set a Z-Wave device configuration parameter.
  description: Allow for changing configuration parameters of your Z-Wave devices.
```

---

same comment as above. How to deal with the developer tools which doesn't understand the target (yet)

---

We won't register this with the entity service helper since we don't want to register the service for any specific platform.

---

The dev tools will understand this on the upcoming beta. All core services have been adjusted already.